dist/logs/logzioLogsFunction/core.js

```diff
@@ -1,60 +1,186 @@
-const logger = require('logzio-nodejs');
-const DataParser = require('./data-parser');
 
-function getCallBackFunction(context) {
-  return function callback(err, bulk) {
-    if (err) {
-      context.err(`logzio-logger error: ${err}`, err);
-      context.bindings.outputBlob = bulk;
+const { ContainerClient } = require('@azure/storage-blob');
+const logger = require('logzio-nodejs');
+const fs = require('fs');
+const util = require('util');
+const appendFileAsync = util.promisify(fs.appendFile);
+const containerName = 'logziologsbackupcontainer';
+const updateTimestamp = log => {
+    if (log.time) {
+        return {
+            ...log,
+            '@timestamp': log.time,
+        };
     }
-    context.done();
-  };
+    return log;
+};
+const updateFolderSize = () => {
+    const stats = fs.statSync(fileName);
+    const fileSizeInKb = stats.size / 1000;
+    folderSize += fileSizeInKb;
+}
+const arrayIsEmpty = array => {
```

Comment:
if you're only checking it once and you already know that you're using an array this is redundant, check directly from the usage (see remark at usage)